src/__tests__/hooks.test.js

```diff
@@ -52,9 +52,9 @@ describe.skip('hooks', () => {
     const testPageData = { page: 'TestPage' };
 
     const TestPage = () => {
-      const { Track } = useTracking({}, { dispatchOnMount: true });
+      const { Track } = useTracking(testPageData, { dispatchOnMount: true });
 
-      return <Track>{null}</Track>; // TODO: what does it mean for the API if this was <Track dispatchOnMount /> instead of useTracking({ dispatchOnMount: true }) above?
+      return <Track>{null}</Track>;
```

Comment:
I _think_ this should still be expected to pass even if you don't return `<Track />` yea?

```suggestion
      return null;
```